templates/index.html

```diff
@@ -14,22 +14,25 @@ <h2>Items</h2>
         {% for item in items %}
         <a href="#" class="list-group-item list-group-item-action">
           <div class="d-flex w-100 justify-content-between">
-            <h5 class="mb-1">{{ item.id }}:   {{ item.title }}</h5>
-            <small>{{ item.status }}</small>
+            <h5 class="mb-1">{{ item.position }}: {{ item.name }}</h5>
+            <small>status: {{ item.status }}</small>
+            <small>description: {{ item.description }}</small>
+            <form action="complete/{{ item.id }}" method="post">
+              <button class="btn btn-outline-primary" type="submit">Complete</button>
+            </form>
+            <form action="undo/{{ item.id }}" method="post">
+              <button class="btn btn-outline-primary" type="submit">Undo</button>
+            </form>
```

Comment:
You could put each of the complete/undo `<form>` elements in `{% if ... %}` blocks so that they only appear when applicable.